Gemfile

```diff
@@ -39,7 +40,7 @@ gem 'font-awesome-sass'
 gem 'jquery-rails' # Use jquery as the JavaScript library
 gem 'momentjs-rails'
 gem 'sass-rails' # Use SCSS for stylesheets
-gem 'uglifier' # Use Uglifier as compressor for JavaScript assets
+gem 'terser' # Alternative to Uglifier
```

Comment:
Did you mean to do this at the same time? Suppose it's fine though if you're happy for it to go

Reply:
I didn't and thought I'd recreated and repushed the branch. Will try again...